tools/cachegrand-benchmark-generator/src/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+include(FindPkgConfig)
+
+pkg_check_modules(LIBPCRE2 REQUIRED libpcre2-8) # apt install libpcre2-8 libpcre2-8-dev
```

Comment:
Once this cmake will be included automatically in the root project, it will be necessary to have these 2 dependencies so it would be necessary to update:
- the build documentation, at least the ubuntu / debian, I can take care of the fedora one later
- the docker image used for the ci build, the one used for the release will not need it as this is not going to be included